### ISSUE TITLE
Hello there))

### ISSUE THREAD

**Issue by FiftyFour7250** [JoshLmao/Genify#14]
Heeeeeey. I'm the guy who asked you to improve site last summer. Just saw that you were back to work, and I noticed that my bug reports hidden and marked as spam (probably because that account was registered to a temp mail :D). I guess they're not available to you too. So if you need examples of the of structure of song titles that don't have lyrics, or have wrong lyrics - let me know. (Yea, still hope you fix it). I also shared about your app yesterday, think now a little more people will start using it, hope this will give you motivation)))

![2324234](https://user-images.githubusercontent.com/78786088/107608519-44f32880-6c4d-11eb-8041-c856dac6f20d.JPG)
![2222](https://user-images.githubusercontent.com/78786088/107608540-50465400-6c4d-11eb-9ece-733c4d21013e.JPG)

**Comment by JoshLmao** [comment]
Hey! Yes, I noticed this a while ago too and I couldn't remember what they said. If you still know what those issues are, I'd appreciate it if you could remake them (just the To Do one). 

And thanks for sharing! I will work on this again soon, just in the middle of Uni work, haha

**Comment by FiftyFour7250** [comment]
Ok. I'm not programmer, and I don't know how it works at all, but it seems to me that it's worth giving up the search in Genius, and use Google search, or another search engine. Because **ALWAYS**, on the query "site:genius.com SongName ArtistName" - **FIRST RESULT** was correct.

"-" issue.
It looks like all the songs that contain " - " in the title can't find lyrics.

![1](https://user-images.githubusercontent.com/78786088/107698669-ef0e9700-6cc5-11eb-843b-0f406df200b6.JPG)
![4](https://user-images.githubusercontent.com/78786088/107699130-ac00f380-6cc6-11eb-8ac8-c44dbcd3ef78.JPG)
![3](https://user-images.githubusercontent.com/78786088/107699016-7c51eb80-6cc6-11eb-8bc0-404c7b855fac.JPG)
![5](https://user-images.githubusercontent.com/78786088/107699243-dce12880-6cc6-11eb-820e-f98f5abeb6ab.JPG)
![9](https://user-images.githubusercontent.com/78786088/107700673-e53a6300-6cc8-11eb-83d3-0d2cdfd2f948.JPG)


"(with...)" issue.
It looks like all the songs that contain " (with...) " in the title can't find lyrics.

![21](https://user-images.githubusercontent.com/78786088/107699672-70b2f480-6cc7-11eb-8eac-f9a17a3c022b.JPG)
![22](https://user-images.githubusercontent.com/78786088/107699823-ac4dbe80-6cc7-11eb-932a-2698dc5ec930.JPG)
![23](https://user-images.githubusercontent.com/78786088/107700212-3a29a980-6cc8-11eb-9cff-b2ce3d7861e7.JPG)
![24](https://user-images.githubusercontent.com/78786088/107700359-6c3b0b80-6cc8-11eb-8fa2-c7e0b26865e5.JPG)
![25](https://user-images.githubusercontent.com/78786088/107700577-b7551e80-6cc8-11eb-95a2-7d45f8b2c461.JPG)


Artist Name issue.
If the artist's names different in Spotify and Genius - app can't find lyrics. Google is still doing great.

No "WWE" in the artists on Genius. 
![32](https://user-images.githubusercontent.com/78786088/107701532-1c5d4400-6cca-11eb-914b-2a15aded47b5.JPG)

This guy has two nicknames, but on Genius all his songs under one name.
![41](https://user-images.githubusercontent.com/78786088/107703157-5596b380-6ccc-11eb-95dd-82421f93fff8.JPG)

Probably "x" and "&"
![42](https://user-images.githubusercontent.com/78786088/107703395-a0b0c680-6ccc-11eb-98ce-83360e9a8985.JPG)


And...

I have no idea why it doesn't works.
![31](https://user-images.githubusercontent.com/78786088/107701286-c2f51500-6cc9-11eb-89bc-14213c721b10.JPG)

Interesting, this is unreleased track that I have in local files, and the name of which I came up with myself - Google still finds it.
![43](https://user-images.githubusercontent.com/78786088/107703749-2b91c100-6ccd-11eb-8502-1c6fa7339d3a.JPG)

**Comment by JoshLmao** [comment]
Thanks for the research. I mean, this primarily comes down to Google's algorithm showing why it's the best. Unfortunately, from a programmers point of view, using Google's API's would limit the sites capabilities immensely. I used it previously for the YouTube video feature but I would quickly hit the API rate limit just by myself. 

I know my algorithm to perform the best search on Genius isn't amazing, but I just need to work towards improving it. I'd also like to avoid adding more than necessary API calls to different services

**Comment by FiftyFour7250** [comment]
Well then I hope you can fix it)
Added another issue. If I come across something else - I'll also add it there. Good luck))

**Comment by FiftyFour7250** [comment]
Wow! You fixed it. Though you wrote that search improved only for remixes, I checked the songs above, checked others, and now everything works perfectly. All lyrics correctly. Thanks)

**Comment by JoshLmao** [comment]
Yeah, glad you noticed! There's still some other scenarios I need to fix, such as songs with "Live" in or "feat.." so I'll leave the issue open but I got "Remix" working. Thanks for your example pics too, I used the songs in them to test 😉

**Comment by FiftyFour7250** [comment]
I found something else. It looks like something with a dot. I don't know if it's worth fixing, especially if it might mess up the algorithm you have now. After all, these cases are not enough. Just letting you know)

Rihanna, Eminem -- Love The Way You Lie (Part II) - Pt. 2 `(lyrics from another song)`
https://open.spotify.com/track/6M6lsQB4OhqL41eld29PeC?si=L8CbzpH1TQG5gZ9_27fBBQ

Kendrick Lamar, Zacari -- LOVE. FEAT. ZACARI.
https://open.spotify.com/track/6PGoSes0D9eUDeeAafB2As?si=DUNZwVmaQU-42tySqHQdzQ

Баста, Скриптонит -- Рай.он  `(lyrics from another song)`
https://open.spotify.com/track/0XEjmpySsQRF2b3cfPAuPP?si=Cq47_4yITxaQ1CnJ_30LNA

Kid Cudi -- Solo Dolo, Pt. III
https://open.spotify.com/track/27oVCAziETRbNuo5A8LNpg?si=7NpZdb07SKeadHIrrm5VUg

**Comment by JoshLmao** [comment]
Any case where Genify doesn't work, I want to attempt to fix and get working so any examples you have, let me know! I appreciate it :D I'll get around to them all eventually, haha

**Comment by FiftyFour7250** [comment]
N-word problem. Even here :D

`DMX -- My Ni**as - Soundtrack Version` (R.I.P.)
https://open.spotify.com/track/1lJpJnrv0m25urQix0oYx3?si=5fB1HniqSRGQDzz23c0snQ

`NAS -- N.I.*.*.E.R. (The Slave and the Master)`
https://open.spotify.com/track/58crvUIMh7VjARw1HAhsU3?si=18gioe03Rhy79xBT33pwsg

`Meek Mill, YG, Snoop Dogg -- That's My N**** (with Meek Mill, YG & Snoop Dogg)` (lyrics from another song)
https://open.spotify.com/track/3i1B1PnsfnOQPGVIzhf1lr?si=j2cmSmJrTtO17Xi5rfTHRg

`JAY-Z, Kanye West -- Ni**as In Paris`
https://open.spotify.com/track/4Li2WHPkuyCdtmokzW2007?si=6EVSlRqASsmLQlxL_R8QzQ

`Lil Wayne -- That's What Ni**as Do`
https://open.spotify.com/track/7nrdN0MyGIbsWbfowJpyJW?si=y6Wen0r2S6yqhRmW_BfT7g


This one works `¯\_(ツ)_/¯`

`NAS -- Y'all My Ni**as`
https://open.spotify.com/track/0w5OlAK2fO54k9b2t2IPfH?si=GfNxikfuTciXE2Oo0ZS7BA
